events/hfe_events_leaders.txt

```diff
@@ -1278,16 +1262,15 @@ country_event = {
 	is_triggered_only = yes
 	trigger = {
 		hfe_trigger_leaders_enabled = yes
-		is_country_type = default
+#		is_country_type = default
```

Comment:
This is for a reason

Reply:
There is no need for this to exist. Provided the person getting this Event is a player, why should a player-controlled Exodus fleet (or anything else) be disqualified?

Reply:
exodus dont need leaders

Reply:
Hence the "or anything else" addition that I placed near the end of my sentence.